### PR TITLE
Clarify the meaning of `remote` when giving NGen image run options

### PR DESCRIPTION
**Pull request by jameshalgren**
### TL;DR
This fixes confusing language in the guide script. Please feel free to suggest something better. 

### More detail
The guide script has a prompt to give the user the option to run with an existing container or to pull a 'fresh' version from the docker repository. 

This commit changes the language in that prompt from a somewhat confusing: 
```
1) Run NextGen Model using local docker image
2) Run Nextgen using remote docker image
3) Exit
```
(aren't all docker images local? what am I getting in a remote image?)

... to the (hopefully) more clear:
```
1) Run NextGen using existing local docker image	      
2) Run NextGen after updating to latest docker image
3) Exit
```

The idea is that the #1 "local" run does not go recompile or rebuild anything, so it might be the best option if you have a setup that is working and you simply want to re-run to reproduce a result (or see a new result from a slightly modified ngen realization, etc.) The #2 "remote" option goes to dockerhub and grabs the latest updates -- useful if you are having problems or trying to test out recently released functionality.

(*Note* -- changed "NextGen Model" to just "NextGen" to reflect the standard language.)

### CODE REVIEW

**Comment by jameshalgren** [comment]
@Reviewers This is a one-liner. Do the lines in the `select-case` statement immediately following also need to be updated?

**Review by benlee0423 — APPROVED** [review]
